README.md

```diff
@@ -284,7 +282,11 @@ The following websites have been built with Volto. You can find their complete s
 - [2023.ploneconf.org](https://github.com/plone/ploneconf.org/tree/2023) - Volto project for [Plone Conference 2023 site](https://2023.ploneconf.org)
 - [plone.org.br](https://github.com/plonegovbr/plone.org.br) - Volto project for the [Brazilian Plone Community](https://plone.org.br)
 - [nsw-design-system-plone6-kit](https://github.com/pretagov/nsw-design-system-plone6-kit) - NSW Design System Plone 6 Kit Volto project for [NSW.gov.au sites](https://digitalnsw.pretagov.com.au/)
+- [volto-eea-design-system](https://github.com/eea/volto-eea-design-system) - EEA Design System Plone 6 Kit Volto project for [European Environment Agency web sites](https://eea.github.io/volto-eea-design-system/)
```

Comment:
@stevepiercy Ops, we have a duplicate here :)

Reply:
Fixed in https://github.com/collective/awesome-volto/pull/20/commits/a7a033c74d866c7d0ec3ae2f8e3f948762ae0443